src/Glpi/Form/Category.php

```diff
@@ -140,6 +140,6 @@ public function setChildren(array $children): void
     #[Override]
     public function getChildren(): array
     {
-        return $this->children;
+        return $this->children ?? [];
```

Comment:
Instead of doing this, you should just define the default value of the property (`private array $children = [];`).